### ISSUE TITLE
original notation incipits

### ISSUE THREAD

**Issue by pe-ro** [music-encoding/music-encoding#481]
There seems to be a lot of confusion about how to encode original notation incipits.  It's not surprising, given that there hasn't been much guidance provided.

I always thought that the original notation incipit material would be handled using a `<section>` element, but that's not working --
1. not everyone "got that memo",
2. the incipit can be editorial (i.e., added by an editor, encoder, etc.) or part of the transcription (i.e., it can exist in the source being transcribed), and
3. an "incipit" as used in the header for bibliographic identification purposes is not the same as an "incipit" which provides an original clef, key signature, and time signature.

So, I think it would be helpful to add a new `<origIncip>` element as the first child of `<score>` with the same content model as `<section>`. Whereas `<incip>` is effectively a separately-render-able score, `<origIncip>` would be rendered with the encoded content of `body/mdiv/score`, with or without a visual space (controlled by `@restart`) between it and the transcribed content.

**Comment by ahankinson** [comment]
Can we keep `<incip>` and use an attribute to control whether it was original or 'supplied'?

**Comment by pe-ro** [comment]
> Can we keep `<incip>` and use an attribute to control whether it was original or 'supplied'?

No, because `<incip>` and `<origIncip>` will have different functions and, therefore, different content.  `<incip>` has the content model --

```xml
<content>
  <rng:zeroOrMore>
    <rng:ref name="model.headLike"/>
  </rng:zeroOrMore>
  <rng:optional>
    <rng:ref name="tempo"/>
  </rng:optional>
  <rng:optional>
    <rng:ref name="role"/>
  </rng:optional>
  <rng:optional>
    <rng:choice>
      <rng:ref name="clef"/>
      <rng:ref name="clefGrp"/>
    </rng:choice>
  </rng:optional>
  <rng:optional>
    <rng:choice>
      <rng:ref name="perfRes"/>
      <rng:ref name="perfResList"/>
    </rng:choice>
  </rng:optional>
  <rng:optional>
    <rng:ref name="key"/>
  </rng:optional>
  <rng:optional>
    <rng:ref name="meter"/>
  </rng:optional>
  <rng:optional>
    <rng:ref name="mensuration"/>
  </rng:optional>
  <rng:zeroOrMore>
    <rng:choice>
      <rng:ref name="model.annotLike"/>
      <rng:ref name="model.graphicLike"/>
      <rng:ref name="model.scoreLike"/>
      <rng:ref name="incipCode"/>
      <rng:ref name="incipText"/>
    </rng:choice>
  </rng:zeroOrMore>
</content>
```
while `<origIncip>` will use --
```xml
<content>
  <rng:zeroOrMore>
    <rng:choice>
      <rng:ref name="model.annotLike"/>
      <rng:ref name="model.appLike"/>
      <rng:ref name="model.divLike"/>
      <rng:ref name="model.editLike"/>
      <rng:ref name="model.graphicPrimitiveLike"/>
      <rng:ref name="model.milestoneLike.music"/>
      <rng:ref name="model.relationLike"/>
      <rng:ref name="model.sectionPart"/>
      <rng:ref name="model.transcriptionLike"/>
    </rng:choice>
  </rng:zeroOrMore>
</content>
```
In other words, `<origIncip>` contains score data and `<incip>` contains metadata.

**Comment by bwbohl** [comment]
@music-encoding/ig-metadata could you please check on the description of how to use `incipit` in the guidelines as described by @pe-ro above? Maybe this should be implemented there?
